src/freedv_api.h

```diff
@@ -49,6 +49,7 @@
 #define FREEDV_MODE_800XA       5
 #define FREEDV_MODE_700C        6
 #define FREEDV_MODE_700D        7
+#define FREEDV_MODE_2020        8
```

Comment:
Would be nice if you also would add the statement 
```
#if !defined(FREEDV_MODE_2020_EN)
        #define FREEDV_MODE_2020_EN FREEDV_MODE_EN_DEFAULT
#endif
```
See a couple of linces below.

Reply:
Thanks, I added that to brad-2020.